sagemaker-pipelines/step-decorator/computer-vision-examples/computer-vision-pipeline.ipynb

```diff
@@ -0,0 +1,1101 @@
+{
```

Comment:
<blockquote>in your S3 bucket under the <code>results</code></blockquote><p> </p><p>Is it <code>f"s3://{bucket_name}/{prefix_project}/{prefix_results}",</code>?</p><p>As the results folder is not directly under S3 bucket, it'd be good to give the complete folder path here.</p><p> </p><p>An alternative is, we can return the entire output s3 path in the evaluate function and retrieve it via:</p><p> </p><pre class="ql-syntax" spellcheck="false">output_file_s3_uri = execution.result(step_name="EvaluateModel")
</pre><p> This way it'd be easier to retrieve the output path</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4543/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Reply:
Great suggestion! I will add a cell which prints the output path for the user.